internal/policy/repository/eventsourcing/policy_age.go

```diff
@@ -0,0 +1,59 @@
+package eventsourcing
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/caos/zitadel/internal/errors"
+	es_models "github.com/caos/zitadel/internal/eventstore/models"
+	"github.com/caos/zitadel/internal/policy/model"
+)
+
+func PasswordAgePolicyQuery(recourceOwner string, latestSequence uint64) *es_models.SearchQuery {
+	return es_models.NewSearchQuery().
+		AggregateTypeFilter(model.PasswordAgePolicyAggregate).
+		LatestSequenceFilter(latestSequence).
+		ResourceOwnerFilter(recourceOwner)
+
+}
+func PasswordAgePolicyAggregate(ctx context.Context, aggCreator *es_models.AggregateCreator, id string, sequence uint64) (*es_models.Aggregate, error) {
+	return aggCreator.NewAggregate(ctx, id, model.PasswordAgePolicyAggregate, projectVersion, sequence)
+}
+
+func PasswordAgePolicyCreateAggregate(aggCreator *es_models.AggregateCreator, project *PasswordAgePolicy) func(ctx context.Context) (*es_models.Aggregate, error) {
+	return func(ctx context.Context) (*es_models.Aggregate, error) {
+		if project == nil {
+			return nil, errors.ThrowPreconditionFailed(nil, "EVENT-kdie6", "project should not be nil")
+		}
+		var err error
+		id, err := idGenerator.NextID()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
When did we decide that? in my opinion we must generate id's in the `CreateAggregate` functions. Because we want the same behaviour also if the functions is not called in this package. 

e.g. create org in admin. the admin package is not reliable to create the id of the org and user it should be done in the `CreateAggregate`-funcs in my opinion

Reply:
If we decide to create the id before we call this function we have to check the id in this function

Reply:
we discussed it again and it should be in eventstore